src/subclass/mod.rs

```diff
@@ -91,7 +91,7 @@
 //! // This is the struct containing all state carried with
 //! // the new type. Generally this has to make use of
 //! // interior mutability.
-//! pub struct SimpleObject {
+//! pub struct SimpleObjectPrivate {
```

Comment:
I'm not a fan of this pattern. We have namespaces in Rust, so let's use those :)

I usually put this into an internal `imp` module, and the public part outside of it. Like in the [`listbox_model.rs`](https://github.com/gtk-rs/examples/blob/pending/src/bin/listbox_model.rs) example, which you could update with these macro changes too

Reply:
I used the same style as https://github.com/gtk-rs/examples/blob/master/src/bin/basic_subclass.rs (and some other places), but I suppose that would work too.

Reply:
Yeah I meant to change that one for a long time already. Thanks for the reminder :)